Example/PeakCoreDataExample/PeakCoreDataExample/EventsTableViewController.swift

```diff
@@ -53,20 +53,19 @@ class EventsTableViewController: UITableViewController, PersistentContainerSetta
     }
     
     @IBAction func deleteButtonTapped(_ sender: Any) {
-        Event.batchDelete(in: viewContext)
+        persistentContainer.performBackgroundTask { backgroundContext in
```

Comment:
Could you update the equivalent on `EventCollectionViewController`, which I'm guessing doesn't work now.

Reply:
Oops, forgot all about that 🤦‍♂

Reply:
Copied the deletion logic to the collection view controller in https://github.com/3squared/PeakCoreData/pull/11/commits/86d5cd5d81219a68d1af0f2e0cb1df9a8a66ab61.